### PR TITLE
[CP-1331][CP-1332] New thread disappear during creating a new message

### PR DESCRIPTION
**Pull request by dkarski**
Jira: [CP-1331] [CP-1332]

**Description**

[CP-1331]: https://appnroll.atlassian.net/browse/CP-1331?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CP-1332]: https://appnroll.atlassian.net/browse/CP-1332?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/messages/components/messages/messages.component.tsx [hunk 1]
```diff
@@ -316,7 +316,7 @@ const Messages: FunctionComponent<Props> = ({
       )
       if (thread) {
         openThreadDetails(thread)
-      } else {
+      } else if (tmpActiveThread === undefined && thread === undefined) {
```
Comment (wojcik92michal) [inline packages/app/src/messages/components/messages/messages.component.tsx:319]: Shouldn't it be covered by some test?

Comment (wojcik92michal) [inline packages/app/src/messages/components/messages/messages.component.tsx:319]: (giving it here, as it is easier to have a thread)

Not sure if this is a scope of this task, but seems to me that when you write a message and then select a contact, the message disappears. Is it expected?

https://user-images.githubusercontent.com/41268731/171599734-0125f1ce-0e6f-4355-b450-c5e1d941385f.mov

Comment (wojcik92michal) [inline packages/app/src/messages/components/messages/messages.component.tsx:319]: (giving it here, as it is easier to have a thread)

Not sure if this is a scope of this task, but when you type a phone number that is assigned to one of your contacts and then send the message, the user is not redirected to the new thread.

https://user-images.githubusercontent.com/41268731/171599969-a8ba74af-0ad2-4639-b0b6-ca015f49d6fb.mov

Comment (dkarski) [reply packages/app/src/messages/components/messages/messages.component.tsx:319]: > Is it expected?

Looks like space for improvement. However,  it isn't the scope of these bugs. Let's cover it as part of the next issue 😉

Comment (dkarski) [reply packages/app/src/messages/components/messages/messages.component.tsx:319]: Can be 😉 Hold my beer 🍺 😅

Comment (dkarski) [reply packages/app/src/messages/components/messages/messages.component.tsx:319]: https://github.com/mudita/mudita-center/pull/747/commits/ecdc99ecee38aea2b7131f1321401ef7a15a7797

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**